dc_utils/templates/dc_forms/field_checkbox.html

```diff
@@ -7,15 +7,15 @@
                 {% for radio in field %}
                     <label class="ds-field-checkbox" for="{{ radio.id_for_label }}">
                         {{ radio.tag }}
-                        <span>{{ radio.choice_label }}</span>
+                        <span>{{ radio }}</span>
                     </label>
                 {% endfor %}
             </div>
         </fieldset>
     {% else %}
         <label class="ds-field-checkbox">
             {{ field }}
-            <span>{{ field.label }}</span>
+            <span>{{ field }}</span>
```

Comment:
I'm not sure I understand the change here. To me it looks like line 17 and 18 are serving the exact same content, except one of them is inside a `<span>` . Is that right?